src/main/webapp/app/exercises/programming/manage/update/programming-exercise-update.component.html

```diff
@@ -19,19 +19,32 @@ <h2 id="jhi-programming-exercise-heading-import" jhiTranslate="artemisApp.progra
                 [isImport]="isImportFromExistingExercise || isImportFromFile"
                 [isLocal]="isLocal"
                 [importOptions]="importOptions"
+                [isSimpleMode]="isSimpleMode()"
+                [isEditFieldDisplayedRecord]="isEditFieldDisplayedRecord()"
+                [courseId]="courseId"
             />
             <hr class="mb-5" />
-            <jhi-programming-exercise-difficulty
+            @if (!isSimpleMode()) {
+                <!-- The mode section would only contain the difficulty in the simple mode,
+                in the simple mode the difficulty will be part of the general section -->
```

Comment:
Would it make sense to still keep it here? This way the transition to easy/hard mode is simpler since the known settings stay at the same place

Reply:
I think that this is a matter of taste. If you do not have a particularly strong opinion here I would leave it like this, as the primary goal right now is to make the simple mode as easy as possible.
For now at least the local storage will remember the latest settings, so if you are using the advanced mode on a device and do not switch it back you will stay in advanced mode - so there should not be much confusion?